cpp/src/io/parquet/page_hdr.cu

```diff
@@ -526,8 +537,13 @@ void __host__ DecodePageHeaders(ColumnChunkDesc* chunks,
 {
   dim3 dim_block(128, 1);
   dim3 dim_grid((num_chunks + 3) >> 2, 1);  // 1 chunk per warp, 4 warps per block
+  // TODO: note we will no longer need this env variable and use_fixed_op
+  // once it's enabled by default.
+  char* opt        = std::getenv("USE_FIXED_OP");
+  int use_fixed_op = (opt == nullptr || opt[0] == '0') ? 0 : (opt[0] == '1' ? 1 : 2);
```

Comment:
IIUC, the monolithic kernel still supports the fixed-width types so we can opt-out of the microkernels. Can we expect any benefit from the removal of the fixed-width path from the monolithic kernel (which I assume we'll do at some point)?